vexim/config/httpheaders.php

```diff
@@ -1,3 +1,4 @@
+<!DOCTYPE html>
```

Comment:
I ran the html code through a code checker to identify problems (of some pages, there are probably more issues). So we would need to add it to each site manually before the `<html>`-tag? I didn't see any impact on rendering with our fancy code ;-)